arbeitszeit_flask/translations/de/LC_MESSAGES/messages.po

```diff
@@ -1251,79 +1227,66 @@ msgstr "Produktion"
 msgid "Language"
 msgstr "Sprache"
 
-#: arbeitszeit_flask/templates/macros/list_coordinations_of_cooperation.html:30
-msgid "Start of coordination"
-msgstr "Beginn der Koordination"
-
-#: arbeitszeit_flask/templates/macros/list_coordinations_of_cooperation.html:32
-msgid "End of coordination"
-msgstr "Ende der Koordination"
+#: arbeitszeit_flask/templates/macros/plan_details.html:15
+msgid ""
+"This plan ID identifies your plan. It is used to account for labour time by "
+"workers and companies that work on, or respectively consume, the planned "
+"product or service."
+msgstr ""
+"Mit Hilfe dieser Plan ID wird dein Plan identifiziert. Sie wird von "
+"Arbeiter*innen und Betrieben genutzt, während sie an der Herstellung eines "
+"Produktes arbeiten bzw. wenn sie ein Produkt konsumieren, um Arbeitszeit zu "
+"verbuchen."
 
-#: arbeitszeit_flask/templates/macros/list_coordinations_of_cooperation.html:41
-msgid "No coordinators found."
-msgstr "Keine Koordinator*innen gefunden."
+#: arbeitszeit_flask/templates/macros/plan_details.html:66
+msgid ""
+"Status can be \"Active\", i.e. plan was approved and current date is within "
+"planning timeframe, or \"Inactive\", i.e. plan is not yet approved or it is "
+"expired."
+msgstr ""
+"Status kann \"Aktiv\", d.h. der Plan wurde genehmigt und das aktuelle Datum "
```

Comment:
I prefered moving "sein" to the beginning: "Status kann "Aktiv" sein... oder "Inaktiv", d.h. der Plan..."

Reply:
done